Utilities/bootstrap

```diff
@@ -555,15 +532,27 @@ def get_swiftpm_flags(args):
             "-Xlinker", "@executable_path/../../../../../SharedFrameworks",
         ])
 
-    # Add a rpath to find core swift libraries on linux and Android. We don't
-    # need a rpath for Darwin because the Swift libraries are present in the OS.
-    if platform.system() == 'Linux':
-        if 'ANDROID_DATA' in os.environ:
-            build_flags.extend(["-Xlinker", "-rpath=$ORIGIN/../lib/swift/android"])
-            # Don't use GNU strerror_r on Android.
-            build_flags.extend(["-Xswiftc", "-Xcc", "-Xswiftc", "-U_GNU_SOURCE"])
-        else:
-            build_flags.extend(["-Xlinker", "-rpath=$ORIGIN/../lib/swift/linux"])
+    # Add a relative rpath to find core Swift libraries on platforms where they
+    # aren't known to be present as part of the base OS.
+    target_info_json = call_output([args.swiftc_path, '-print-target-info'], verbose=args.verbose)
```

Comment:
Instead of shelling out again, can you compute this information at start of the script and store in `args`? We pass  that around everywhere.

Reply:
Will do.

Reply:
Ah. We can just remove it since SwiftPM doesn't deploy that far back on Darwin anymore.